qiling/os/windows/structs.py

```diff
@@ -115,6 +117,35 @@ class PEB(Struct):
 
     return PEB
 
+class NT_TIB(struct.BaseStruct):
+    '''
+    _NT_TIB structure
+
+    Below output is from Windows RS4
+    0: kd> dt _NT_TIB
+    nt!_NT_TIB
+        +0x000 ExceptionList    : Ptr64 _EXCEPTION_REGISTRATION_RECORD
+        +0x008 StackBase        : Ptr64 Void
+        +0x010 StackLimit       : Ptr64 Void
+        +0x018 SubSystemTib     : Ptr64 Void
+        +0x020 FiberData        : Ptr64 Void
+        +0x020 Version          : Uint4B
+        +0x028 ArbitraryUserPointer : Ptr64 Void
+        +0x030 Self             : Ptr64 _NT_TIB
+    '''
+
+    _fields_ = (
+        ('ExceptionList',           ctypes.c_void_p),
```

Comment:
Here and on other new structures:
Using types like `c_void_p` causing the structure definition to be affected by the hosting system architecture, and not by the emulated system - as it should be. For example, if a 64-bit Windows binary is emulated on a 32-bit host, `c_void_p` will be 4 bytes long instead of the expected 8. For that reason we use `native_type` and `pointer_type` that are chosen according to the emulated arch.

Also, for simplicity we use simple types instead of pointers. ctypes pointers are assigned in a slightly different way, which makes it a bit more complicated, and since we do not reference 'real' memory here, we keep it simple. This is why we use `pointer_type` that is actually `native_type`.